config/main_window.cpp

```diff
@@ -69,11 +69,18 @@ MainWindow::MainWindow(QWidget *parent)
     mSortFilterProxyModel = new QSortFilterProxyModel(this);
 
     mSortFilterProxyModel->setSourceModel(mDefaultModel);
+    mSortFilterProxyModel->setFilterKeyColumn(-1);
+    mSortFilterProxyModel->setFilterCaseSensitivity(Qt::CaseInsensitive);
     actions_TV->setModel(mSortFilterProxyModel);
 
     mSelectionModel = new QItemSelectionModel(actions_TV->model());
     actions_TV->setSelectionModel(mSelectionModel);
 
+    connect(filter_LE, &QLineEdit::textChanged, [this]{
+      auto filterTerm = filter_LE->text();
+      mSortFilterProxyModel->setFilterRegularExpression(filterTerm);
+    });
+
```

Comment:
Glad to see such a fast response!

Your work seems good to me but please change the above connection to:

```c++
connect(filter_LE, &QLineEdit::textChanged, mSortFilterProxyModel, &QSortFilterProxyModel::setFilterWildcard);
```

Many users aren't familiar with regex and may see strange results when, for example, they type `alt*`. And there's no need to a lambda.